File: FlexServ Backend/SpringBoot/AuthService/src/main/java/com/flexserv/dto/response/ServiceProviderResponseDto.java
package com.flexserv.dto.response;

import lombok.Data;
import java.util.Set;

@Data
public class ServiceProviderResponseDto {
    private Long id;
    private Long userId;
    private String userName;
    private String userEmail;
    private String companyName;
    private Integer experienceYears;
    private String bio;
    private Boolean isVerified;
    private Double rating;
    private Boolean companyAvailable;
    private Set<String> offeredServices;
}

File: FlexServ Backend/DotNet/AuthService/Dtos/ServiceProviderResponseDto.cs
using System.Collections.Generic;

namespace AuthService.Dtos
{
    public class ServiceProviderResponseDto
    {
        public long Id { get; set; }
        public long? UserId { get; set; }
        public string? UserName { get; set; }
        public string? UserEmail { get; set; }
        public string CompanyName { get; set; } = string.Empty;
        public int? ExperienceYears { get; set; }
        public string? Bio { get; set; }
        public bool? IsVerified { get; set; }
        public double? Rating { get; set; }
        public bool? CompanyAvailable { get; set; }
        public HashSet<string> OfferedServices { get; set; } = new HashSet<string>();
    }
}
